docker-compose.yml

```diff
@@ -0,0 +1,55 @@
+version: "3"
+
+networks:
+  frontend:
+    external: true
+  app:
+    driver: bridge
+    internal: false
```

Comment:
Is it intentional that this file should be usable in general or only as a part of the Azure infrastructure? I would expect the former since it is now placed in the root of the repository in but if I try to run `docker-compose up` I get the following error:

```
11:29 $ docker-compose up
WARNING: The COMPOSE_PROJECT_NAME variable is not set. Defaulting to a blank string.
WARNING: The COMPOSE_DOMAIN variable is not set. Defaulting to a blank string.
Creating network "material-list_app" with driver "bridge"
ERROR: Network frontend declared as external, but could not be found. Please create the network manually using `docker network create frontend` and try again.
```

If it is for general use and require special handling then please update the documentation accordingly. Otherwise I think it belongs in the infrastructure folder.

Reply:
It not related to kubernetes it's was intended for development and was based on ITK docker composer setup. Removed it as it is not needed and ITK is not going to do development on this project.